electron/lib/d3.phylogram.js

```diff
@@ -224,7 +224,7 @@ phylogram.build = function(selector, nodes, options = {}) {
 			.size([h, w])
 			.sort(node => (node.children ? node.children.length : -1))
 			.children(options.children || (node => node.branchset))
-
+			console.log(tree)
```

Comment:
Do we still need to log this "tree" here?

Reply:
whoops, that should be removed.